CHANGELOG.md

```diff
@@ -25,6 +25,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Added `Ctrl-Space` hotkey for triggering interactive code completions.
 
+- More mobile friendly completion items via media query
+
+### Fixed
+
+- Pressing ESC to close completions menu doesn't de-focus the code-editor
+
+- Word wrapping on long completion item details doesn't extend to multiple lines
+
+- Elevation on completion items list matches other elevations on project
+
```

Comment:
For new release needs to be moved under an `## Unreleased`. I can move as part of release.

Reply:
Ah sorry. That would be great thanks!